generators/entity/prompts.js

```diff
@@ -237,6 +238,45 @@ function askForRelationsToRemove() {
     }.bind(this));
 }
 
+function askForTableName() {
+    // don't prompt if there are no relationships
+    var entityTableName = this.entityTableName;
```

Comment:
we already have `this.entityTableName = _.snakeCase(this.options['table-name'] || this.name).toLowerCase();`  at the constructor of this subgen and hence it should never be undefined at this point. Are you finding it undefined in any case?